recipes/pyboat/meta.yaml

```diff
@@ -0,0 +1,73 @@
+{% set name = "pyboat" %}
+{% set version = "0.7.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 546c81d1cb952e25cd7660642741530e12c3b5053bacd54c7e745c4eae8becee
+
+build:
+  number: 0
+  entry_points:
+    - pyboat = pyboat:main  
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ cdt('mesa-libgl-devel') }}  # [linux]
+    - {{ cdt('mesa-dri-drivers') }}  # [linux]
+    - {{ cdt('libselinux') }}  # [linux]
+    - {{ cdt('libxdamage') }}  # [linux]
+    - {{ cdt('libxxf86vm') }}  # [linux]
+    - {{ cdt('libxext') }}     # [linux]
+    
+  host:
+    - matplotlib >=3.1
```

Comment:
Do you need all of these? I'd except `python` and `pip` to be enough

Reply:
Ok, let me try!